SalesforceCommerceCloud/pwa-kit
# Fix heading hierachy and aria-hidden icons

# Description



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- Check out code
- Run Retail App
- Confirm that aria-hidden is present in the my account log out
![image](https://github.com/user-attachments/assets/e1297c20-d292-4fe5-8bbf-1055c8cde68a)
![image](https://github.com/user-attachments/assets/a11c45bc-3c70-4068-8e52-dd00f515b26f)


For heading hierachy, since we have a lot of content/links inside the footer. The straightforwards way to fix the issue is to separate footer as another `section`. To check there is no more heading hierachy violation. 
- Run the Retail App
- Login and go to my account page
- Run axe tools, and you should not see any headings errors since headings in footer is now considered headings of a new section and separate from the main content. 

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
